waiter/bin/ci/run-integration-tests-composite-scheduler.sh

```diff
@@ -21,6 +21,15 @@ KITCHEN_DIR=${TEST_APPS_DIR}/kitchen
 NGINX_DIR=${TEST_APPS_DIR}/nginx
 SEDIMENT_DIR=${TEST_APPS_DIR}/sediment
 
+# set SAML authenticator variables
```

Comment:
Should these variables also move inside the if to restrict when they get initialized?

Reply:
done

Reply:
they are needed for the authenticator that's configured in the .edn file even if the integration test doesn't run

Reply:
okay